contrib/crio-wipe/lib.bash

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+set -eu
+
+get_major() {
+    echo "$@" | grep -Po '^(\"|crio version )?\K\d(?=\.\d+\..*)'
+}
+
+get_minor() {
+    echo "$@" | grep -Po '^(\"|crio version )?\d\.\K\d+(?=\..*)'
+}
+
+perform_wipe() {
+    if [[ $WIPE -eq 0 ]]; then
+        echo "Wiping storage"
+        rm -rf $CONTAINERS_STORAGE_DIR
+    else
+        echo "Spoofing storage wipe"
+    fi
+    exit 1
+}
+
+check_versions_wipe_if_necessary() {
+    # $1 should be the new version
+    # $2 should be the old version
+
+    # if we didn't find the old version. Our version file is invalid. Trigger wipe
+    if [[ -z ${2+x} ]]; then
```

Comment:
I dunno about the `+x` bash magic.  In any case, should probably make sure neither `$1` and `$2` are empty.

Reply:
@cevich this is actually to make sure when $1 and $2 are empty it doesn't crash the program because of set -u, but instead gracefully quit. I got it from the internet :)

Reply:
Okay well it must be foolproof and I'm the fool :smile: